vm/interpreter/src/default_runtime.rs

```diff
@@ -654,12 +654,13 @@ where
     ) -> Result<Randomness, ActorError> {
         let r = if rand_epoch > networks::UPGRADE_HYPERDRIVE_HEIGHT {
             self.rand
-                .get_chain_randomness_v2(personalization, rand_epoch, entropy)
+                .get_chain_randomness(personalization, rand_epoch, entropy)
                 .map_err(|e| e.downcast_fatal("could not get randomness"))?
         } else {
-            self.rand
-                .get_chain_randomness_v1(personalization, rand_epoch, entropy)
-                .map_err(|e| e.downcast_fatal("could not get randomness"))?
+            panic!("FVM doesn't support older networks")
+            // self.rand
+            //     .get_chain_randomness_v1(personalization, rand_epoch, entropy)
+            //     .map_err(|e| e.downcast_fatal("could not get randomness"))?
```

Comment:
if we are using `BACKEND=native` are the older versions supported? I thought they were still, is that not the case?

Reply:
Only nv14 is supported in this branch - regardless of which backend is used. The native backend could possibly sync from older network versions but not as far back as genesis.

Reply:
I see, we can prioritize adding the previous versions to the FVM then, since that's handy to have